core/src/main/scala/org/apache/spark/deploy/rest/StandaloneRestClient.scala

```diff
@@ -147,9 +148,15 @@ private[deploy] class StandaloneRestClient extends Logging {
     conn.setRequestProperty("Content-Type", "application/json")
     conn.setRequestProperty("charset", "utf-8")
     conn.setDoOutput(true)
-    val out = new DataOutputStream(conn.getOutputStream)
-    out.write(json.getBytes(Charsets.UTF_8))
-    out.close()
+    var out: DataOutputStream = null
```

Comment:
Er, right...I had looked at this and not sure why I thought it needed to be a var...will change.